content/05.discussion.md

```diff
@@ -1,8 +1,45 @@
-# Discussion
-Here mention why performnace increases in the beginning for the generative model then decreases
+## Discussion
 
-Discuss discriminator model performance given generative model
-
-Mention Take home messages
-
-1. have a centralized set of negative label functions and focus more on contstructing positive label functions
+We tested the feasibility of re-using label functions to extract relationships from literature.
+Through our sampling experiment we found that adding relevant label functions increases prediction performance.
+This can been seen in the on-diagonals of generative model performance plots (Figure {@fig:generative_model_auroc} and {@fig:generative_model_auprc}).
+An interesting finding is performance increases when text patterns and domain heuristic label functions are added on top of the distant supervision label functions. 
+This increase happens because curated databases have low coverage and end up missing information.  
```

Comment:
Do you have evidence for this? If so, I missed it. If not, it's probably best to delete it.